### PR TITLE
progress: Add package

### PR DESCRIPTION
**Pull request by nikil**
Maintainer: me
Compile tested: ar71xx, x86_64
Run tested: x86_64

Description: Coreutils Progress Viewer (formerly known as 'cv')

Signed-off-by: Nikil Mehta nikil.mehta@gmail.com

### CODE REVIEW

File: utils/progress/Makefile [hunk 1]
```diff
@@ -0,0 +1,52 @@
+#
+# Copyright (C) 2006-2016 OpenWrt.org
```
Comment (diizzyy) [inline utils/progress/Makefile]: Please don't use OpenWrt.org copyright unless you are under contract.
https://www.lede-project.org/faq.html#who_holds_the_copyright_of_lede_code

File: utils/progress/Makefile [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+PKG_LICENSE_FILES:=LICENSE
+
+include $(INCLUDE_DIR)/package.mk
+
+define Package/progress
+  SECTION:=utils
+  CATEGORY:=Utilities
+  TITLE:=Coreutils Progress Viewer (formerly known as 'cv')
+  URL:=https://github.com/Xfennec/progress
+  DEPENDS:=+libncurses
```
Comment (diizzyy) [inline utils/progress/Makefile]: +libncursesw ?

**Comment by diizzyy** [comment]
The tarball name is no-go, you probably need to pull the source in this case. I'm not aware of any way to rename downloaded tarballs in a sane matter.

**Comment by nikil** [comment]
Thanks again for the comments.  I've changed this package to pull from git - let me know if I set it up right (it compiles and tests OK).

File: utils/progress/Makefile [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+
+PKG_NAME:=progress
+PKG_VERSION:=0.13.1
+PKG_RELEASE:=1
+
+PKG_SOURCE_PROTO:=git
+PKG_SOURCE_URL:=https://github.com/Xfennec/progress.git
+PKG_SOURCE_VERSION:=f6894b827c638af5cf8c5c622e66491c179ba2eb
+PKG_SOURCE_SUBDIR:=$(PKG_NAME)-$(PKG_VERSION)
+PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION)-$(PKG_SOURCE_VERSION).tar.gz
```
Comment (diizzyy) [inline utils/progress/Makefile]: xz here please :-)

Comment (nikil) [inline utils/progress/Makefile]: Done, thanks.

**Comment by diizzyy** [comment]
@thess @hnyman 
This looks like a reasonable package to the repo?

https://www.howtoforge.com/tutorial/how-to-monitor-progress-of-linux-commands-using-pv-and-progress-utilities/
So as long it's "local" it seems to do it's job from what I can tell.

**Pull request merged**